src/CommandLine/Core/Verb.cs

```diff
@@ -12,12 +12,17 @@ sealed class Verb
         private readonly string name;
         private readonly string helpText;
         private readonly bool hidden;
+        private readonly bool isDefault;
 
-        public Verb(string name, string helpText, bool hidden = false)
+        public Verb(string name, string helpText, bool hidden = false, bool isDefault = false)
         {
-            this.name = name ?? throw new ArgumentNullException(nameof(name));
+            if (!isDefault && string.IsNullOrWhiteSpace(name))

```

Comment:
//name should not be null or empty independent of isDefault
```suggestion
            if ( string.IsNullOrWhiteSpace(name))
```